mathics/builtin/numbers/numbertheory.py

```diff
@@ -99,8 +99,8 @@ def apply(self, n, evaluation):
         "Divisors[n_Integer]"
         if n == Integer0:
             return None
-        return Expression(
-            SymbolList, *sympy.divisors(n.to_sympy()), element_conversion_fn=from_sympy
+        return to_expression(
+            SymbolList, *sympy.divisors(n.to_sympy()), elements_conversion_fn=from_sympy
```

Comment:
`elements_conversion_fn=Integer`

Reply:
We may be able to use Expression here.... checking...

Reply:
You are correct we can use Integer here. And we can use `n.value` since we know n is an Integer. 

It also appears that the elements will always be sorted because that's how sympy.divisors works. 
But to_expression will figure that out. It has to make a pass over the arguments anyway so checking for sortedness comes pretty much for free.

As noted elsewhere though all of this will get replaced by something like 

```
ListExpression(*sympy_divisors(n.value), Integer)
```

and by doing this we can save both forms of the Mathics list as well as a Python/SymPy/mpmath/numpy - convenient form too.